Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp

```diff
@@ -428,10 +425,30 @@ String CDMInstanceOpenCDM::sessionIdByInitData(const InitData& initData, bool fi
 
 bool CDMInstanceOpenCDM::isSessionIdUsable(const String& sessionId) const
 {
-    std::string sessionIdAsStdString(sessionId.utf8().data());
-    const auto& element = m_sessionsMap.find(sessionIdAsStdString);
+    auto session = lookupSession(sessionId);
+    return session && (session->lastStatus() == media::OpenCdm::KeyStatus::Usable);
```

Comment:
I know it can give you the creeps, but the parenthesis after the && are not needed.